src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
I dont think we want to check in the `application.properties` file itself, just `application.properties.EXAMPLE`